src/ScheduleService.Business/Workspace/GetWorkspaceCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using LT.DigitalOffice.ScheduleService.Business.Workspace.Interfaces;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Workspace;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Business.Workspace
```

Comment:
In another PR https://github.com/LT-Students/ScheduleService/pull/1/files put ; after namespace